.github/ISSUE_TEMPLATE.md

```diff
@@ -10,9 +10,9 @@ Please answer the following questions for yourself before submitting an issue.
 
 #### Environment
 
-* DART version: 6.3.0 / master
-* OS name and version name(or number): Ubuntu 14.04 / Windows 10 / macOS Sierra
-* Compiler name and version number: GCC 4.8.5 / Clang 3.9.1
+* DART version: Select master / 6.3.0 / other (please specify)
```

Comment:
This might be excessively nitpicky, but I was thinking more like `Select DART version: master / 6.3.0 / other (please specify)`

Reply:
It sounds more nature! Updated.